### PR TITLE
RAD-231 Use maven license header plugin

### PR DESCRIPTION
**Pull request by soniro**
## Description
Added the maven license plugin. Therefore removed the {{clear_blank_lines_in_javadoc_comment}} formatter rule and added/reformatted missing licenses in all java/xml files.

Obviously, no test was written.

## Related Issue
see https://issues.openmrs.org/browse/RAD-231

## Checklist:
- [x] My pull request only contains one single commit.
- [x] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by soniro** [comment]
Sorry, GitHub added all 3 commits.
Please let me know if this is a problem.

File: omod/src/test/resources/RadiologyOrderSearchHandlerComponentTestDataset.xml [hunk 1]
```diff
@@ -1,11 +1,13 @@
-<?xml version='1.0' encoding='UTF-8'?>
```
Comment (teleivo) [inline omod/src/test/resources/RadiologyOrderSearchHandlerComponentTestDataset.xml]: @sonira why is it deleting this line here? It has left it in the other testdataset xmls

**Comment by teleivo** [comment]
@soniro its ok, I think I can squash em on merging. The only thing is this one XML dataset I commented, where the first line is deleted. Wondering why?

**Comment by soniro** [comment]
@teleivo Oh this was not on purpose. Sorry for that.

**Comment by teleivo** [comment]
@soniro thanks! its working well when I do "mvn clean install" tells me if there is a license header missing somewhere. the only thing I am having trouble with is executing the plugin manually maybe you can help me out here :)

I added
```
  <pluginGroups>
    <pluginGroup>org.openmrs.maven.plugins</pluginGroup>
    <pluginGroup>com.mycila</pluginGroup>
  <pluginGroups>
```

to my m2/settings.xml and when I do

```
mvn license:check
```

it downloads the maven-metadata.xml for the license plugin
```
<?xml version="1.0" encoding="UTF-8"?>
<metadata>
  <plugins>
    <plugin>
      <name>license-maven-plugin</name>
      <prefix>license</prefix>
      <artifactId>license-maven-plugin</artifactId>
    </plugin>
  </plugins>
</metadata>
```

but then I get the error

No plugin found for prefix 'licence' in the current project and in the plugin groups [org.openmrs.maven.plugins, com.mycila, com.spotify, org.apache.maven.plugins, org.codehaus.mojo]

do you have a suggestion for me?

**Comment by soniro** [comment]
Mmh... For me it works without problems, when I run `mvn license:check` or `mvn license:check -pl api` (for a specific submodule) within my openmrs-module-radiology folder.

I haven't add anything to my settings.xml.

**Comment by teleivo** [comment]
@soniro mhmm, ok Ill figure it out. Thanks!!

**Pull request merged**